test/unittests/execute_test.cpp

```diff
@@ -1,21 +1,37 @@
 #include "execute.hpp"
 #include "limits.hpp"
 #include "parser.hpp"
+#include <gmock/gmock.h>
 #include <gtest/gtest.h>
 #include <test/utils/asserts.hpp>
 #include <test/utils/hex.hpp>
 
 using namespace fizzy;
 
+MATCHER(Trapped, "")
```

Comment:
following other matchers in gmock
```suggestion
MATCHER(Traps, "traps")
```